css/elements.css

```diff
@@ -16,6 +16,11 @@ body {
   overflow: hidden;
 }
 
+#spec-container > emu-clause {
```

Comment:
This won't work for documents like [ecma-402](https://tc39.es/ecma402/), which use `emu-import` to organize their source and consequently are laid out like `#spec-container > emu-import > emu-clause`.

A hacky (but completely adequate) fix would be to just add that selector to this bit. Other solutions are possible.